src/containers/ToDoList/ToDoList.jsx

```diff
@@ -54,6 +60,8 @@ export const ToDoList = () => {
   ]);
   const [newTaskInput, setNewTaskInput] = useState("");
   const [filteringType, setFilteringType] = useState();
+  const [isIconChanging, setIsIconChanging] = useState(false);
```

Comment:
Do you really need this? Couldn't the type of the button be based on whether `isDarkTheme` true or false? It seems like both these vars are keeping exact same information.